RELEASE-NOTES.md

```diff
@@ -10,6 +10,10 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 ## [Unreleased] - Release date yyyy-mm-dd
 
 ### Added
+- Added option to quest's `SignedDistance` class and C API to toggle whether
+  the distance query computes the sign.
+- Added batched option to accelerate quest's signed distance query example.
```

Comment:
I think the statement about 'batched option' should be clarified --> what this means is that one can pass in a vector of point coordinates and process them in parallel, or something along those lines.

Reply:
I'll try to rework this.